.github/workflows/proof_ci.yaml

```diff
@@ -0,0 +1,188 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: MIT-0
+# CBMC starter kit 2.9
+name: Run CBMC proofs
+on:
+  push:
+    branches-ignore:
+      - gh-pages
+  pull_request:
+    branches-ignore:
+      - gh-pages
+  workflow_dispatch:
+
+# USAGE
+#
+# If you need to use different versions for tools like CBMC, modify this file:
+# .github/workflows/proof_ci_resources/config.yaml
+#
+# If you want the CI to use a different GitHub-hosted runner (which must still
+# be running Ubuntu 20.04), modify the value of this key:
+# jobs.run_cbmc_proofs.runs-on
+
+jobs:
+  run_cbmc_proofs:
+    runs-on: cbmc_ubuntu-latest_64-core
```

Comment:
just curious, where does this customer runner `cbmc_ubuntu-latest_64-core` come from?

Reply:
This is a [Large Runner](https://docs.github.com/en/actions/using-github-hosted-runners/using-larger-runners) pool that we have access to. This pool is shared with all other CBMC-using repositories in the awslabs GitHub organisation and has a certain concurrency limit. I'll ping you offline with more details.